lib/embed/viewer/common_viewer.rb

```diff
@@ -228,7 +228,12 @@ def height_style
       end
 
       def width_style
-        "max-width:#{width}px;" if width
+        "max-width:#{with_style_attribute};"
+      end
+
+      def with_style_attribute
+        return '100%' unless width
+        "#{width}px"
```

Comment:
should this be `width_style_attribute`?  seems like a plausible intentional name, but also seems like it could be a typo?